### PR TITLE
Introduce new component examples of adding common metadata objects in…

### PR DESCRIPTION
**Pull request by topcon-pconlon**
…cluding delays, regions, materials, operators and auth codes to sites. Include a new workflow example that creates a new site and configures it with sufficient metadata to allow Topcon Hauler apps to connect and haul materials between regions.

### CODE REVIEW

File: components/metadata/metadata_create/operator_create/operator_create.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    logging.basicConfig(format=args.log_format, level=args.log_level)
+    # << Arguments
+
+    server = ServerConfig(a_environment=args.env, a_data_center=args.dc)
+   
+    logging.info("Running {0} for server={1} dc={2} site={3}".format(os.path.basename(os.path.realpath(__file__)), server.to_url(), args.dc, args.site_id))
+
+    token = get_token(a_client_id=args.oauth_id, a_client_secret=args.oauth_secret, a_scope=args.oauth_scope, a_server_config=server)
+
+    operator_rdm_bean = OperatorMetadataTraits.post_bean_json(a_first_name=args.operator_first_name, a_last_name=args.operator_last_name, a_code=args.operator_code)
```
Comment (topcon-rballinger) [inline components/metadata/metadata_create/operator_create/operator_create.py:53]: Move this to separate function to match the pattern you've used elsewhere?

File: components/metadata/metadata_create/task_create/task_create.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+                    "surface":{"locked":False}
+                },
+        "materials":{},
+        "name":a_task_name,
+        "sequenceTypeClass":"none",
+        "_rev":str(uuid.uuid4()),
+        "_type":"sl::task",
+        "_id": str(uuid.uuid4()),
+        "_at":at
+    }
```
Comment (topcon-rballinger) [inline components/metadata/metadata_create/task_create/task_create.py:38]: add to metadata_traits?

File: components/sites/site_create.py [hunk 3]
```diff
@@ -13,83 +13,97 @@
...
+            "lon": float(a_longitude)
+        },
+        "contact": {
+            "phone": a_phone,
+            "email": a_email,
+            "name" : a_name
+        },
+        "name"     : a_site_name,
+        "timezone" : a_timezone
+    }
```
Comment (topcon-rballinger) [inline components/sites/site_create.py:57]: metadata_traits?

Comment (topcon-pconlon) [reply components/sites/site_create.py:57]: Agreed

File: components/metadata/metadata_create/operator_create/operator_create.py (hunk 1 above)
Comment (topcon-pconlon) [reply components/metadata/metadata_create/operator_create/operator_create.py:53]: Good find.

**Review by topcon-rballinger — APPROVED** [review]

**Pull request merged**